docs/patterns/existing-eks-observability-accelerators/existing-eks-nginx-observability.md

```diff
@@ -184,7 +184,7 @@ EXTERNAL_IP=your-nginx-controller-external-ip
 5. Start some sample NGINX traffic by entering the following command.
 ```
 SAMPLE_TRAFFIC_NAMESPACE=nginx-sample-traffic
-curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/master/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/sample_traffic/nginx-traffic/nginx-traffic-sample.yaml |
+curl https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/k8s-deployment-manifest-templates/nginx/nginx-traffic-sample.yaml |
 sed "s/{{external_ip}}/$EXTERNAL_IP/g" |
```

Comment:
This wont work if you point to external ip of ingress, you should be pointing to DNS name of the LB, thats why recommended you to grab the DNS of the LB Nginx service and that should be passed on as external ip. Also its host and not external ip.

Reply:
```
export EXTERNAL_IP=$(kubectl get svc blueprints-addon-nginx-ingress-nginx-controller -n nginx-ingress-sample --output jsonpath='{.status.loadBalancer.ingress[0].hostname}')
```

Reply:
Use this command.